heron/tools/common/src/python/utils/config.py

```diff
@@ -382,20 +382,35 @@ def defaults_cluster_role_env(cluster_role_env):
 ################################################################################
 # Parse the command line for overriding the defaults
 ################################################################################
-def parse_override_config(namespace):
-  """Parse the command line for overriding the defaults"""
+def parse_override_config_and_write_file(namespace):
+  """
+  Parse the command line for overriding the defaults and
+  create an override file.
+  """
+  overrides = parse_override_config(namespace)
   try:
     tmp_dir = tempfile.mkdtemp()
     override_config_file = os.path.join(tmp_dir, OVERRIDE_YAML)
     with open(override_config_file, 'w') as f:
-      for config in namespace:
-        f.write("%s\n" % config.replace('=', ': '))
+      for k, v in overrides.items():
+        f.write("%s:%s\n" % (k, v))
```

Comment:
there is a space after `:`, which breaks our jeckins ci for two days. @maosongfu  @nwangtw